quill-sql/src/main/scala/io/getquill/context/QueryExecution.scala

```diff
@@ -570,8 +570,9 @@ object PrepareDynamicExecution {
       matchingExternals: List[External],
       secondaryLifts: List[Planter[_, _, _]] = List()
   ): Either[String, (List[Planter[_, _, _]], List[Planter[_, _, _]])] = {
-    val encodeablesMap =
```

Comment:
I don't think that `encodeableMap` and `secondaryEndoablesMap` should be declared here, this could be move after the declaration of the `enum UidStatus` closer to where they are used.